### PR TITLE
add basic JavaDoc

### PR DESCRIPTION
**Pull request by X1nto**
Added some basic documentation, not fully complete but at least it gives some hints about what APIs do.

### CODE REVIEW

File: Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java [hunk 1]
```diff
@@ -51,21 +51,52 @@
...
             this.username = username;
             this.send = send;
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
+         * @param send Whether to send the message or not. If false, messages will appear locally, otherwise they'll be sent to the current channel.
+         * @param username Username for Clyde. Requires <code>send</code> to be false.
+         * @param avatarUrl Avatar URL for Clyde, must be raw. Requires <code>send</code> to be false.
```
Comment (Vendicated) [inline Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:98]: what does 'must be raw' mean

File: Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java [hunk 2]
```diff
@@ -51,21 +51,52 @@
...
+         * @param content Output message content
+         * @see CommandResult#CommandResult(String, List, boolean)
+         */
         public CommandResult(String content) {
             this(content, null, true);
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
```
Comment (Vendicated) [inline Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:71]: is this really true? has someone tried what happens if u try to sned embeds? xD

File: Aliucord/src/main/java/com/aliucord/api/PatcherAPI.java [hunk 3]
```diff
@@ -76,6 +114,9 @@ public void run() {
         return _unpatch;
     }
 
+    /**
+     * Unpatches all patches.
```
Comment (Vendicated) [inline Aliucord/src/main/java/com/aliucord/api/PatcherAPI.java:118]: Removes all patches would probably sound better ngl

**Review by Vendicated — COMMENTED** [review]
docs docs docs docs

File: Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java (hunk 2 above)
Comment (Juby210) [reply Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:71]: it is locked in aliucord api

Comment (Vendicated) [reply Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:71]: oh okay thats good

File: Aliucord/src/main/java/com/aliucord/api/PatcherAPI.java (hunk 3 above)
Comment (X1nto) [reply Aliucord/src/main/java/com/aliucord/api/PatcherAPI.java:118]: Indeed, it sounds better

File: Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java (hunk 1 above)
Comment (X1nto) [reply Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:98]: Should be an actual path to the file and not a redirect, as redirects don't seem to work

Comment (Vendicated) [reply Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:98]: so it cant be a proxy url? raw makes me think of github raw links so its kinda confusing xD maybe change it to "must be a direct link, not a redirect"

Comment (X1nto) [reply Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java:98]: Yep, sounds better

**Review by Vendicated — APPROVED** [review]

**Review by Juby210 — APPROVED** [review]

**Pull request merged**